### ISSUE TITLE
Changing API version of Clients

### ISSUE THREAD

**Issue by jerameel** [Shopify/shopify-node-api#338]
# Issue summary

Write a short description of the issue here ↓
Changing API version of Clients (Admin GraphQL, Storefront) without relying on context.

## Expected behavior

What do you think should happen?
Should be able to override/change api version used on clients using additional parameter

## Actual behavior

What actually happens?
Cannot change api version without initializing shopify context

Tip: include an error message (in a `<details></details>` tag) if your issue is related to an error

## Steps to reproduce the problem

1. Use api client without context, apiversion defaults to unstable
1.
1.

## Reduced test case

The best way to get your bug fixed is to provide a [reduced test case](https://webkit.org/test-case-reduction/).

---

## Checklist

- [ ] I have described this issue in a way that is actionable (if possible)

**Comment by shopsideau** [comment]
Yes would really love to see this functionality to set a specific version on the client (or as a param when initialising it).

**Comment by JayAdra** [comment]
Is it possible to get this looked at/added soon? Crucial feature for us, would love to see it implemented.

**Comment by shopsideau** [comment]
A workaround I found that seems to be working so far is just manually specifying the version after the import, i.e.

```js
Shopify.Context.API_VERSION = '2022-04';
```

This is without using `Context.initialize` anywhere too. An official solution would be preferred, but hopefully this helps for now.

**Comment by jessicazu** [comment]
@shopsideau 
Great! It is a simple solution and works. Thank you very much.
